migrate.py

```diff
@@ -148,58 +191,93 @@ def fail(msg: str) -> NoReturn:
                 print(f"[Dry Run] {old_path} would be moved to {new_path}")
             else:
                 fsmove(old_path, new_path)
+        elif command == "changeLink":
+            # Update media file path and hash for the "changeLink" option
+            # The intention of this option is to change the Windows path \ sign to Linux's / sign
+            media_files: List[Tuple[str, str]] = cursor.execute(
+                "SELECT id, path from media_file WHERE path LIKE ? || '%'",
+                OLD_QUERY_ARGS,
+            ).fetchall()
```

Comment:
Small comment, but rather than check windowsToLinuxPath and vice versa in the loop, could you assign the replacement strings outside? Something like:
```python
# General
path_strings: List[str] = []

# if change link
path_strings = ["\\", "/"] if windowsToLinuxPath else ["/", "\\"]

# for other changes
if path_strings:
    new_track_path.replace(path_strings[0], path_strings[1])
```
Or something like that